Sources/App/Core/Twitter.swift

```diff
@@ -73,22 +71,47 @@ extension Twitter {
         return preamble + description + link
     }
 
+    static func newPackageMessage(packageName: String,
+                                  repositoryOwner: String,
+                                  url: String,
+                                  summary: String?) -> String {
+        createMessage(preamble: "New package: \(packageName) by \(repositoryOwner)",
+                      summary: summary,
+                      url: url)
+    }
+
+    static func versionUpdateMessage(repositoryOwner: String,
+                                     repositoryName: String,
+                                     url: String,
+                                     version: SemanticVersion,
+                                     summary: String?) -> String {
+        createMessage(preamble: "\(repositoryOwner) just released \(repositoryName) v\(version)",
```

Comment:
As I mentioned in #799, I think we want `packageName` here as well, instead of `repositoryName`. Agree to change that as part of this PR?

Reply:
Yep!